src/shared/components/mutationTable/column/FunctionalImpactColumnFormatter.tsx

```diff
@@ -59,315 +56,232 @@ class FunctionalImpactColumnTooltip extends React.Component<
-                        </tr>
                     </tbody>
                 </table>
             </div>
         );
     }
 
-    public static mutationAssessorText() {
-        return (
-            <div style={{ width: 450, height: 130 }}>
-                Mutation Assessor predicts the functional impact of amino-acid
-                substitutions in proteins, such as mutations discovered in
-                cancer or missense polymorphisms. The functional impact is
-                assessed based on evolutionary conservation of the affected
-                amino acid in protein homologs. The method has been validated on
-                a large set of disease associated and polymorphic variants (
-                <a href="https://www.ncbi.nlm.nih.gov/clinvar/" target="_blank">
-                    ClinVar
-                </a>
-                ).
-                <br />
-                <b>
-                    Mutation Assessor V4 data is available in the portal since
-                    Oct. 8, 2024.
-                </b>{' '}
-                New manuscript is in progress. Click{` `}
-                <a href="http://mutationassessor.org/r3/" target="_blank">
-                    here
-                </a>
-                {` `} to see information about V3 data.
-            </div>
-        );
-    }
-
-    public static siftText() {
-        return (
-            <div style={{ width: 450, height: 100 }}>
-                <a href={Sift.SIFT_URL} target="_blank">
-                    SIFT
-                </a>{' '}
-                predicts whether an amino acid substitution affects protein
-                function based on sequence homology and the physical properties
-                of amino acids. SIFT can be applied to naturally occurring
-                nonsynonymous polymorphisms and laboratory-induced missense
-                mutations.
-            </div>
-        );
-    }
-
-    public static polyPhen2Text() {
-        return (
-            <div style={{ width: 450, height: 100 }}>
-                <a href={PolyPhen2.POLYPHEN2_URL} target="_blank">
-                    PolyPhen-2
-                </a>{' '}
-                (Polymorphism Phenotyping v2) is a tool which predicts possible
-                impact of an amino acid substitution on the structure and
-                function of a human protein using straightforward physical and
-                comparative considerations.
-            </div>
-        );
-    }
-
     public render() {
         return (
             <div>
-                {this.state.active === 'mutationAssessor' &&
-                    FunctionalImpactColumnTooltip.mutationAssessorText()}
-                {this.state.active === 'sift' &&
-                    FunctionalImpactColumnTooltip.siftText()}
-                {this.state.active === 'polyPhen2' &&
-                    FunctionalImpactColumnTooltip.polyPhen2Text()}
+                {this.state.active ===
```

Comment:
Should we use mobx here instead of react state? I think we used to avoid built-in react state in favor of mobx. Not sure if we are still following that practice though.